### ISSUE TITLE
Support for OCamldoc

### ISSUE THREAD

**Issue by bobbypriam** [314eter/atom-ocaml-merlin#35]
There might be something that I'm missing, but I can't seem to get the docs showing on autocompletion even though [this package seems to support it](https://github.com/314eter/atom-ocaml-merlin/blob/master/lib/merlin.coffee#L121-L122). So far I've tried with OCaml's standard library, Jane Street's Core, and my own functions but I only get type info, not docs.

![screen shot 2017-04-08 at 11 49 13 pm](https://cloud.githubusercontent.com/assets/4920820/24830742/16e805cc-1cb6-11e7-8981-3a9a5be93f61.png)

Are there any config or setup that I need to do to get it working?

Also related: It might also be helpful if there was a "Show Documentation" feature (like "Toggle Type" etc.) for identifier under cursor.

**Comment by Kakadu** [comment]
Yeah, documentation will be very helpful. It seems that it has been introduced in [merlin 3.x ](https://github.com/ocaml/merlin/blob/master/doc/dev/PROTOCOL.md#document--position-position---identifier-string-).

**Comment by 314eter** [comment]
It's in older versions of merlin too (https://github.com/ocaml/merlin/blob/master/doc/dev/OLD-PROTOCOL.md#completion). I don't know why it doesn't work.

**Comment by bobbypriam** [comment]
I see. Let me see if I could find the time to look into it.
